k8s-install/scripts/clean.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
```

Comment:
This script looks pretty neat.  Do you think this will likely be a more commonly used script - if so, I think it could benefit from beefing up the comments a little - especially in a lot of the nested routines.  The code itself looks fine, but a few more comments would go a long way.  That said, it's probably because my shell scripting is a bit rusty that I'm so slow reviewing  :-)

Reply:
Thanks. I think long term, script will disappear. But I am going to add more comments anyway.